.codeclimate.yml

```diff
@@ -1,6 +1,8 @@
 engines:
   duplication:
     enabled: true
+    exclude_fingerprints:
+    - 871d85232f90c108abf687a789b06606
```

Comment:
@kelseyq https://codeclimate.com/github/SexualHealthInnovations/django-wizard-builder/wizard_builder/models.py#issue_5819027bde98af0001000024

(link was in the description, was easy to miss)